python/cloudtik/runtime/spark/scripts/configure.sh

```diff
@@ -390,13 +390,11 @@ function configure_jupyter_for_spark() {
 }
 
 function configure_local_hdfs_fs() {
-    # Nothing to do now
-    :
+    sudo sed -i 's?#user_allow_other?user_allow_other?' /etc/fuse.conf
 }
 
 function configure_hdfs_fs() {
-    # Nothing to do now
-    :
+    sudo sed -i 's?#user_allow_other?user_allow_other?' /etc/fuse.conf
```

Comment:
If the configure will be the same for local hdfs, let's write a util function configure_fuse_options so that configure_local_hdfs_fs and configure_hdfs_fs call this function to share the code.